### DISCUSSION TITLE
[M-Z] (22 Gennaio 2019) Esercizio 2

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#433] [Esami & Esoneri M-Z]
## Traccia

<img width="932" alt="image" src="https://github.com/user-attachments/assets/0b9ce88c-6e71-4a95-9287-c1ddb160eda5">

>**Stesso esercizio:** #424 (15 Gennaio 2018 - Esercizio 2)

**Comment by rimaout** [comment]
<img width="600" alt="image" src="https://github.com/user-attachments/assets/d4a5e12b-892d-4a9c-9aad-37dae8273db3">

**Comment by matypist** [comment]
Questo esercizio è un duplicato di [[M-Z] (15 Gennaio 2018) Esercizio 2 #424](https://github.com/sapienzastudentsnetwork/metodi-matematici-per-l-informatica/discussions/424) , per evitare una frammentazione delle proposte di soluzione potete quindi scrivere/leggere eventuali possibili soluzioni direttamente lì
